packages/collector/test/tracing/messaging/bull/test.js

```diff
@@ -70,17 +69,14 @@ mochaSuiteFn.only('tracing/messaging/bull', function () {
       await retry(async () => {
         const spans = await agentControls.getSpans();
 
-        // 1 x bull send?
-
         // TODO: all other bull tests also produce a huge number of spans
```

Comment:
@abhilash-sivan Comment can be moved to e.g. line 232? 

> TODO: https://jsw.ibm.com/browse/INSTA-15029

Reply:
yes. done